packages/block-editor/src/autocompleters/link.js

```diff
@@ -10,12 +10,10 @@ import { decodeEntities } from '@wordpress/html-entities';
 
 const SHOWN_SUGGESTIONS = 10;
 
-/** @typedef {import('@wordpress/components').WPCompleter} WPCompleter */
```

Comment:
Removing this as WPCompleter was never exported from the components package until https://github.com/WordPress/gutenberg/pull/67410, where it was advised that it shouldn't be exported.